bin/k8s-deploy-secrets

```diff
@@ -43,8 +43,9 @@ echo "Deploying Encrypted Secrets"
 for SOPS_SECRET_FILE in "${SOPS_SECRET_FILES[@]}"
 do
   set -x
+  sops ${SOPS_OPTIONS} --decrypt "${SOPS_SECRET_FILE}" | kubeval
   sops ${SOPS_OPTIONS} --decrypt "${SOPS_SECRET_FILE}" | \
-  kubectl replace "--namespace=${NAMESPACE}" -f -
+    kubectl replace "--namespace=${NAMESPACE}" -f -
```

Comment:
@robscott this extra space seems not correct

Reply:
Yeah it is weird, I was trying to indicate that this line is a continuation of the line above and not a standalone command, but I can revert that change.

Reply:
@lanceops Just reverted the extra indentation.

Reply:
lgtm